src/app/components/Filters/FilterTabs.tsx

```diff
@@ -0,0 +1,49 @@
+import Umrah from '../../../../public/images/Fillter//Umrah.svg'
+import Flight from '../../../../public/images/Fillter/flight.svg'
+import Hotel from '../../../../public/images/Fillter/Hotel.svg'
+import Tour from '../../../../public/images/Fillter/Tour.svg'
+import Visa from '../../../../public/images/Fillter/Visa.svg'
```

Comment:
Can't we refactor these images by adding all the icons from a single component ? (as suggested before)
- Make svg image to React component (a component which will return svg icon) 
- Use icons from `react-icons` as much as you can. Last option will be creating a custom component